packages/sync-actions/src/product-types.js

```diff
@@ -13,7 +13,8 @@ function createProductTypeMapActions(
   mapActionGroup: (
     type: string,
     fn: () => Array<UpdateAction>
-  ) => Array<UpdateAction>
+  ) => Array<UpdateAction>,
+  config: Object
```

Comment:
I wanted to use `types/sdk` here, but I get flow errors, so I went back to just `Object`
I could use some help to introduce the following:

```
export const SyncActionConfig = {
  ourFlag: Boolean,
}
```

Reply:
What lint errors did you get here?

Reply:
forget it, I effed up